Description: Debianize
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 entrypoint2 (0.0.5-1) unstable; urgency=low
 .
   * source package automatically created by stdeb 0.6.0+git
Author: Andrew King (No comment) <aking1012.com@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- entrypoint2-0.0.5.orig/entrypoint2.egg-info/PKG-INFO
+++ entrypoint2-0.0.5/entrypoint2.egg-info/PKG-INFO
@@ -1,4 +1,4 @@
-Metadata-Version: 1.0
+Metadata-Version: 1.1
 Name: entrypoint2
 Version: 0.0.5
 Summary: easy to use command-line interface for python modules, fork of entrypoint
--- entrypoint2-0.0.5.orig/entrypoint2.egg-info/SOURCES.txt
+++ entrypoint2-0.0.5/entrypoint2.egg-info/SOURCES.txt
@@ -4,6 +4,7 @@ README.rst
 TODO
 pavement.py
 requirements.txt
+setup.cfg
 setup.py
 docs/api.rst
 docs/conf.py
